test/unit-test/CMakeLists.txt

```diff
@@ -127,8 +127,8 @@ foreach( file ${TCP_INCLUDES} )
     endif()
 
     # Use this tool to process all conditional declarations.
-    if(${MODIFIED_FILE} STREQUAL "FreeRTOS_Routing" )
-        execute_process( COMMAND unifdefall -U${Guard} -USEND_REPEATED_COUNT -UpdTRUE_SIGNED -UFreeRTOS_htonl -D__COVERITY__ -DTEST -DipconfigUSE_IPV6 -I ${MODULE_ROOT_DIR}/tools/CMock/vendor/unity/src
+    if(${MODIFIED_FILE} STREQUAL "FreeRTOS_Routing" OR ${MODIFIED_FILE} STREQUAL "FreeRTOS_IP_Private" )
```

Comment:
Why is this needed?

Reply:
`FreeRTOS_Routing.h` and `FreeRTOS_IP_Private.h` needs to be parsed with `ipconfigUSE_IPV6 `and `ipconfigUSE_RA` enabled in order to make certain structures defined inside those headers compatible with the current test case requirements.